src/basis/utils/assign.js

```diff
@@ -0,0 +1,34 @@
+// ES6 Object.assign polyfill
+// jscs:disable
+
+if (!Object.assign) {
+    Object.defineProperty(Object, 'assign', {
```

Comment:
In my case it worked (because in my case vendor lib uses only two arguments for `Object.assign`). But I can't guarantee that other libraries won't use `Object.assign` in a wrong way because the difference between `basis.object.extend` and  `Object.assign` becomes real when `Object.assign` gets more than two arguments
https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/Object/assign